conan/tools/files/files.py

```diff
@@ -263,6 +264,47 @@ def chdir(conanfile, newdir):
         os.chdir(old_path)
 
 
+def chmod(conanfile, path:str, read:bool, write:bool, execute:bool, recursive:bool=False):
+    """
+    Change the permissions of a file or directory. The same as the Unix command chmod, but simplified.
+    It skips in case running on Windows.
+
+    :param conanfile: The current recipe object. Always use ``self``.
+    :param path: Path to the file or directory to change the permissions.
+    :param read: If ``True``, the file or directory will have read permissions for any user.
+    :param write: If ``True``, the file or directory will have write permissions for any user.
+    :param execute: If ``True``, the file or directory will have execute permissions for any user.
+    :param recursive: (Optional, Defaulted to ``False``) If ``True``, the permissions will be applied
+    """
+    if platform.system() == "Windows":
+        conanfile.output.debug("Skipping tools.files.chmod on Windows.")
+        return
+
+    if not os.path.exists(path):
+        raise ConanException(f"Could not change permission: Path \"{path}\" does not exist.")
+
+    def _change_permission(it_path:str):
+        mode = os.stat(it_path).st_mode
+        permissions = [
+            (read, stat.S_IRUSR | stat.S_IRGRP | stat.S_IROTH),
```

Comment:
I am not sure the default should be setting the permissions for everyone, but just for the current user.
Recipes in ConanCenter set ``S_IXUSR/S_IXEXEC`` only, not all.

Reply:
I know some recipes also do ``os.chmod(filename, os.stat(filename).st_mode | 0o111)``, but maybe this has to be checked with @jcar87 too

Reply:
Ok, looking better now, still I wonder why the ``read`` permission will be different to the others, is it not enough that it has "user" read permissions (if for some reason it didn't have it)?

Reply:
Done, only changed owner permission.